notebooks/optimal-stopping-point/osp_model.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    parameters_passing = passing_dist[passing_dist["Distribution Names"] == best_dist][
</pre><p>Can you explain why we get the passing test parameters using the best distribution (best_dist) returned from the failing tests above <code>best_dist, parameters_failing = best_distribution(failure_dist, failures_r)</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/445/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>We chose to get the best_distribution from the failing test distribution because mainly trying to get an optimal stopping point for the failing test.</p>

Reply:
Ah, got it. Good catch. Lemme fix this!

Reply:
Thanks 👍

Reply:
/hold for this

Reply:
<p>Added an explanation here for this.</p>